src/order-history/OrderHistoryPage.jsx

```diff
@@ -180,15 +185,15 @@ class OrderHistoryPage extends React.Component {
         </h1>
         {loadingError ? this.renderError() : null}
         {loaded && hasOrders ? (
-          <React.Fragment>
+          <>
```

Comment:
Just curious, why leave an empty tag here?

Reply:
The linter said so / I think all snippets need to be enclosed within some sort of HTML tag. Since there isn't a standard HTML tag (e.g. div/span/p/etc.) here, we can use empty tags instead.

Reply:
Seems to me like the sort of thing `<div>` is for, but if it works and the linter says it's cool no need to change it.

Reply:
I'm hesitant to add an HTML tag since it could potentially affect the styling/other frontend behaviors -- this was done to fix an linting issue so I'd rather not make any substantive changes here. And from what I gather, empty tags are fairly standard in React.

Reply:
Huh, I can't seem to get useful search results or further reading to learn more about how these fit with HTML syntax and when or why to use them.

Reply:
<React.Fragment> wouldn't actually show in the DOM, and <> is a concise version of writing the same

Reply:
Ah, it's a React thing and not "really" html; more here: https://reactjs.org/docs/fragments.html

Stylistically, I think the linter rule is set the wrong way, since the empty tag-brackets appear strange and it's nearly impossible to get more information about them, whereas  `<React.Fragment>` explicit and easy to learn more about; obviously that's way outside the scope of this PR.